snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/parseex1.cs

```diff
@@ -21,13 +21,15 @@ public static void Main()
             Console.WriteLine("The string to be parsed is null.");   
```

Comment:
line 17 and 18
var newGuid = Guid.Parse(stringGuid);
Console.WriteLine($"Converted {stringGuid} to a Guid");

Reply:
Done.